scripts/install.sh

```diff
@@ -40,14 +40,14 @@ echo
 read -rd '' addToPathDirectivesZSH << EOF
 * Add \$HOME/.local/bin to PATH to access hzc from any directory
   Append the line below in your .zshrc file if it doesn't exists
-  export PATH=\$HOME/.local/bin:\$PATH" >> \$HOME/.zshrc
+  export PATH=\$HOME/.local/bin:\$PATH >> \$HOME/.zshrc
```

Comment:
The message "Append the line below in your .zshrc file if it doesn't exists" is a bit confusing. The user may add `export PATH=\$HOME/.local/bin:\$PATH >> \$HOME/.zshrc` to their zshrc instead of running that line.

Reply:
Yes, thanks for catching that. Will fix it as discussed